statesman-server/src/main/java/io/appform/statesman/server/ingress/IngressHandler.java

```diff
@@ -114,7 +123,9 @@ public boolean invokeEngineForOneShot(String ivrProvider, IngressCallback ingres
         }
         val date = new Date();
         val dataObject = new DataObject(mapper.createObjectNode(), wfTemplate.getStartState(), date, date);
-        wfp.saveWorkflow(new Workflow(wfId, wfTemplate.getId(), dataObject, new Date(), new Date()));
+        val workflow = new Workflow(wfId, wfTemplate.getId(), dataObject, new Date(), new Date());
+        wfp.saveWorkflow(workflow);
+        workflowInitPostProcessing(workflow);
```

Comment:
It's probably better to generate different observable events of a workflow from the workflow db access layer?